lib/order.rb

```diff
@@ -0,0 +1,65 @@
+require_relative "customer.rb"
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
```

Comment:
Similar to the above: `products`, `customers`, and `fullfillment_status` do not need to be `attr_accessor`